src/player.js

```diff
@@ -2601,6 +2605,20 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * Sets the video track selected by the user.
+   * @returns {void}
+   * @private
+   */
+  _setDefaultVideoTrack() {
+    const selectedVideoTrack = this._getVideoTracks()
+      .sort((track1: VideoTrack, track2: VideoTrack) => track2.bandwidth - track1.bandwidth)
+      .find((track: VideoTrack) => track.height && track.height === this._playbackAttributesState.videoHeight);
```

Comment:
what about comparing width ? or label ?

Reply:
I also think we should save the entire object of the track instead of just height, and handle the specific matching algorithm(if to match by width, height or label) afterwards in the `_setDefaultVideoTrack` method

Reply:
done. first compare by label if not found then compare by height